blog/content/post/2022-09-21-qubes-os-summit-2022.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Qubes OS summit 2022 - Summary
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/image-file.png
+author: norbert.kaminski
+layout: post
+published: true
+date: 2022-09-21
+archives: "2022"
+
+tags:
+  - Qubes
+  - Conference
+categories:
+  - Firmware
+  - IoT
+  - Miscellaneous
+  - OS Dev
+  - App Dev
+  - Security
+  - Manufacturing
+
+---
+
+## Long story short
+
+Three weeks ago 3mdeb with Qubes OS team had organized next edition of the
```

Comment:
We organized it with Invisible Things Lab and order should be switched.

Reply:
Fixed.